tmt/options.py

```diff
@@ -8,11 +8,11 @@
 # Verbose, debug and quiet output
 verbose_debug_quiet = [
     click.option(
-        '-v', '--verbose', is_flag=True,
-        help='I want to see more details. Please, be verbose.'),
+        '-v', '--verbose', is_flag=True, multiple=True,
+        help='Show more details. Use multiple times to raise verbosity.'),
```

Comment:
Would be nice to know how many v I can use, same for d

Reply:
I'd say three levels should be enough. At least for the start ;-) But in general, I think we can extend later as/if needed. I'd suggest to document this when have more examples.

Reply:
Let's have it uncapped, and $verbose be INT, Then every plugin decides for itself.. `-v=3` would be the same as `-vvv`.

Or `-ddd`, I don't incline to any specfics, but I'd love to leave the implementation open (not hardcoded) from the start.

Reply:
I've already updated the pull request to store as `int`. I'm fine with keeping it without a max level for now.